yt/frontends/enzo/misc.py

```diff
@@ -13,3 +13,29 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 
+import numpy as np
+
+from yt.utilities.physical_ratios import \
+    rho_crit_g_cm3_h2, cm_per_mpc
+
+def cosmology_get_units(hubble_constant, omega_matter, box_size,
```

Comment:
I'm trying to remember -- does this cover *all* enzo cosmology possibilities, or only flat?  (I know that there are edge cases in enzo's cosmology units, but I'm just checking to see if they are fully covered.)

Reply:
This covers all cosmologies in terms of unit conversions. You might be thinking of the conversions between redshift and time, for which only flat cosmologies are covered with the old enzo functionality. I have [this open PR](https://github.com/yt-project/yt/pull/1714) for converting those functions to a numerically integrated table (as is now done in Enzo), and those will cover all cosmologies. This is unrelated to that, though.

Reply:
Whoops, thanks!